integration/spark-common/src/main/scala/org/apache/spark/sql/execution/streaming/CarbonAppendableStreamSink.scala

```diff
@@ -127,7 +127,7 @@ class CarbonAppendableStreamSink(
         className = sparkSession.sessionState.conf.streamingFileCommitProtocolClass,
         jobId = batchId.toString,
         outputPath = fileLogPath,
-        isAppend = false)
+        false)
```

Comment:
no need to modify this

Reply:
in spark2.2.1 default argument name is "isAppend" and in 2.3.0 it is dynamicPartitionOverwrite. So it is required change